CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define dicotic games, games `x` where both players can move from
+every nonempty subposition of `x`. We prove that these games are small, and relate them
+to infinitesimals.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
```

Comment:
```suggestion
- Define infinitesimal games as games `x` such that `∀ y : ℝ, 0 < y → -y < x ∧ x < y`
```
(bikeshedding: maybe we want to use some other variable name for reals? like `r` perhaps)